package.json

```diff
@@ -21,6 +22,7 @@
     "gulp-rename": "^1.4.0",
     "gulp-yaml": "^2.0.4",
     "handlebars": "^4.3.0",
+    "husky": "^7.0.4",
```

Comment:
This doesn't really work if you don't install husky manually. In order to get this working, you need to include prepare/postInstall script in  `scripts` section of package.json

https://blog.typicode.com/husky-git-hooks-autoinstall/

```
 "scripts": {
    "postinstall": "husky install"
  }
```

Reply:
Added postinstall script